server/openai/openaiClient.js

```diff
@@ -5,9 +5,10 @@ let openai
 const configuration = new Configuration({
   apiKey: process.env.OPEN_AI_API_KEY || 'add-alt-key-here',
```

Comment:
Hiya could you put this in `config.js`, the one in the folder called `config`?

Reply:
will do